config/operator-bundle/hive-clusterserviceversion.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: hive-operator-0.0.1
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: "false"
+    description: |-
+      OpenShift cluster provisioning and management at scale.
+    containerImage: quay.io/dgoodwin/hive:latest
+    createdAt: 2019-02-25T14:29:00Z
+    support: Devan Goodwin
+spec:
+  displayName: Hive
+  description: |
+    OpenShift cluster provisioning and management at scale.
+  keywords: ['kubernetes', 'openshift', 'multi-cluster', 'cluster']
+  version: 0.0.1
```

Comment:
@jmelis it looks like it's valid here to do 0.0.1-HASH here. What would be easiest for your pipeline in this regard?

Maybe a make csv target that builds this file up with the version = 0.0.1-HASH, replaces the container images with a hash tag as well? Your pileline can run that, then publish the resulting bundle with operator-courier?